backend/API/appsettings.json

```diff
@@ -6,7 +6,7 @@
     }
   },
   "AllowedHosts": "*",
-  "ConnectionStrings": {
-    "RookiesConnectionString": "Server=localhost\\SQLEXPRESS;Database=TheRookiesDB;Trusted_Connection=True;"
-  }
+    "ConnectionStrings": {
+        "RookiesConnectionString": "Server=localhost\\SQLExpress;Database=TheRookiesDB;Trusted_Connection=True;"
```

Comment:
As I mentioned before. This setting will be used on Production Env. Then you should defined this with connection to Azure instead.
Besides, Add another connection for Dev Env if needed